### PR TITLE
Fix line-ends and remove trailing whitespace

### PR DESCRIPTION
**Pull request by keithc-ca**

### CODE REVIEW

**Comment by Mesbah-Alam** [comment]
This PR also seems to fail in the Line Endings Check : https://ci.eclipse.org/openj9/job/PullRequest-LineEndingsCheck-Systemtest/175/console

**Comment by keithc-ca** [comment]
Yes, I noticed that. I'm trying to figure out why pull requests for this repo work differently as compared to, say, openj9.

**Comment by keithc-ca** [comment]
I expect the problem is this line in [`.gitattributes`](https://github.com/eclipse-openj9/openj9-systemtest/blob/master/.gitattributes#L2):
```
* text eol=lf
```
I updated that file as well, but as yet it doesn't seem to be helping.

**Comment by keithc-ca** [comment]
Jenkins line endings check

**Comment by keithc-ca** [comment]
@AdamBrousseau Any theories why the line-end check is failing even after removing the 'eol' specification from `.gitattributes` (like openj9)?

**Comment by keithc-ca** [comment]
Does the check require a good version of `.gitattributes` on the main branch?

**Comment by AdamBrousseau** [comment]
Looks like it's complaining about the bat file.
```
ERROR - should have CRLF line terminators: 'openj9.build/scripts/openj9-systemtest-clone-make.bat' type: 'ASCII text'
```

**Comment by keithc-ca** [comment]
Ok, I needed to commit the change locally to `.gitattributes` before committing the change to the `*.bat` file: the check has now passed.

File: .gitattributes [hunk 1]
```diff
@@ -1,7 +1,24 @@
...
+# Exception [1] and GNU General Public License, version 2 with the
+# OpenJDK Assembly Exception [2].
+#
+# [1] https://www.gnu.org/software/classpath/license.html
+# [2] http://openjdk.java.net/legal/assembly-exception.html
+#
+# SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
 
-# Overrides for files to be left untouched - treat as binary
+# Specify files should be considered be binary.
```
Comment (pshipton) [inline .gitattributes:21]: There are missing words here or something, it doesn't read very well.

Comment (keithc-ca) [reply .gitattributes]: Agreed; fixed.

**Review by pshipton — APPROVED** [review]

**Pull request merged**